pom.xml

```diff
@@ -54,24 +54,24 @@
     <dependency>
       <groupId>org.powermock</groupId>
       <artifactId>powermock-api-mockito</artifactId>
-      <version>1.6.5</version>
+      <version>1.7.4</version>
```

Comment:
:no_entry_sign: Before this can be merged it should be tested with test-gen (on our regression tests, but also testing it on one of the benchmarks would be useful). I remember we had this upgraded to 1.6.6 but then had to downgrade back to 1.6.5 (as far as I remember it was to work around a bug in Powermock but I can't find it documented anywhere.. which is my fault as I made the commit):
https://github.com/diffblue/deeptest-utils/commit/30e57b59e2d4c3e1884aa5b496a0e21dd0bb851d#diff-600376dffeb79835ede4a0b285078036

Also making any upgrades here needs to be reflected in platform, test-gen or wherever else we use the dependency I guess.

Reply:
I vaguely remember an issue with 1.6.6, yes. I commented on it here: https://github.com/diffblue/platform/commit/9108c30cbcc2ffc55a0e0d552911f60b23f96a71

I think it was a 1.6.6 specific regression, and in later versions it is fixed. We really should have documented it better.

Reply:
There are also lots of compatibility issues between Mockito and Powermock. Only certain versions are compatible, and this is not well documented. So any changes we make to the versions need to be tested on test-gen.

Reply:
I've made a [branch on test-gen](https://github.com/diffblue/test-gen/compare/rjmunro/latest-dtu) that upgrades to this sub-module. Travis is testing it here:
https://travis-ci.com/diffblue/test-gen/builds/126585889